paddle/fluid/pybind/op_function_generator.cc

```diff
@@ -23,45 +23,47 @@
 #include "paddle/fluid/pybind/pybind.h"
 #include "paddle/fluid/string/string_helper.h"
 
+// clang-format off
```

Comment:
Why disable clang-format?

Reply:
clang-format changes the format of `template string` and the format of generated functions, which we do not want.